Google Analytics V2 Cloud Extract  doc (#933)
rudderlabs/rudderstack-docs: docs/sources/extract/google-analytics-v2.mdx

    - **Service Account Key Authentication**: Enter the service account key in JSON format in the **Service Account JSON Key field**. Refer to the [Google documentation](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys) for more information on obtaining the service account key.

    <img src="../../assets/cloud-extract-sources/ga-oauth-settings-4.png" alt="Google Analytics credentials" />

The rest of the settings are as follows:
<img src="../../assets/cloud-extract-sources/ga-oauth-settings-2.png" alt="Google Analytics credentials" />
<img src="../../assets/cloud-extract-sources/ga-oauth-settings-3.png" alt="Google Analytics credentials" />

- **Replication Start Date**: Choose the start date from which you want RudderStack to ingest the Google Analytics data. RudderStack will not replicate any data before this date.
- **View ID**: Enter the ID of the Google Analytics View you want to fetch the data from. Refer to this [documentation](https://docs.acquia.com/customer-data-platform/connectors/web-analytics-provider/google-analytics-profileid/) for more information on obtaining the View ID.

> - **Custom Reports (Optional)**: Use this field to sync your [custom reports](https://support.google.com/analytics/answer/1033013?hl=en) from Google Analytics. Click **Add Report** and enter the **Report Name**, **Metrics**, and **Dimensions** which RudderStack will add as a JSON array, for example: `{"name": string, "dimensions": [string], "metrics": [string]}`.

Reviewer: They can also edit an existing report or delete. They can add many reports. 
Also I don't think they need to know the underneath implementation of json array.

Author: Thanks for highlighting this, @am6010!
I have added these in the doc.

---
title: "Google Analytics V2"
description: Step-by-step guide on ingesting data from Google Analytics into RudderStack.
aliases: ["Google Analytics Extract Source", "Singer Google Analytics"]
---

> [Google Analytics](https://analytics.google.com/analytics/web/#/) is one of the popular analytics service that allows you to track and report your website traffic across a variety of sources.

Reviewer 2: ```suggestion
[Google Analytics](https://analytics.google.com/analytics/web/#/) is a popular analytics service that lets you track and report your website traffic across a variety of sources.
```

This document guides you in setting up Google Analytics as a source in RudderStack. Once configured, RudderStack automatically ingests your Google Analytics data and routes it to your specified data warehouse destination.

<div class="warningBlock">
All the Cloud Extract sources support sending data only to a <Link to="/destinations/warehouse-destinations/">data warehouse destination</Link>.
</div>

<GhBadge
  label={'Stability'}
  message={'Beta'}
  color={'blueviolet'}
  logo={'github'}
/>

## Getting started

To set up Google Analytics as a Cloud Extract source in RudderStack, follow these steps:

1. Log into your [RudderStack dashboard](https://app.rudderstack.com/).
2. Go to **Sources** > **New source** > **Cloud Extract** and select **Google Analytics V2** from the list of sources.
3. Assign a name to your source and click **Continue**.

### Connection settings

Next, configure the following dashboard settings:

> - **Credentials**: Choose from the two authentication types as explained below:

Reviewer 2: ```suggestion
- **Credentials**: Choose from one of the following two authentication types:
```

>     - **Authenticate via Google (Oauth)**: Authenticate your Google Analytics account via Google (Oauth) by clicking

Reviewer 2: ```suggestion
    - **Authenticate via Google (Oauth)**: Authenticate your Google Analytics account via Google (OAuth) by clicking
```
